packages/nektar/package.py

```diff
@@ -83,7 +80,8 @@ def hasfeature(feature):
         args.append("-DNEKTAR_USE_SCOTCH=%s" % hasfeature("+scotch"))
         args.append("-DNEKTAR_USE_PETSC=OFF")
         args.append("-DNEKTAR_ERROR_ON_WARNINGS=OFF")
-        args.append("-DNEKTAR_USE_MKL=%s" % hasfeature("+mkl"))
+        args.append("-DNEKTAR_USE_MKL=%s" % hasfeature("%intel-oneapi-mkl"))
```

Comment:
@cmacmackin Should this be `^intel-oneapi-mkl` instead?

Reply:
Yes, you are right.

Reply:
That is now fixed.